CIP-Liesenfelt-Shelleys_Voltaire_decentralization_update/README.md

```diff
@@ -164,41 +196,39 @@ Large differences between k-parameter and the k-effective of the network represe
-4. The a0 input parameter must require, not incentivize, pool operators to pledge to support community delegation and sybil protection.
+3. There should be a very clear cause and effect relationship between the $(a0,k)$ input parameters and the resulting K-effective decentralization.
+4. The $a0$ input parameter must require, not incentivize, pool operators to pledge to support community delegation and sybil protection.
 5. The decentralization result will be quantified in terms of group decentralization of block production, not stake saturation relative to pools.
 6. Implementation should be smooth, easy, clear, and beneficial for all stakeholders and operators.
 7. A new reward equation should be computationally simple and elegant.
 
 ## The Proposed Reward Formula
 
-The proposed reward retains the function of k for limiting rewards based on stake but repurposes the a0 parameter for enforcing reward limits based on pledge leverage. The equation equally balances both reward parameters. Instead of a0 ranging from 0.0 to infinity the a0 parameter is intended to range from 10,000.0 down to 1.0. An a0 value of 100.0 would require pools to pledge 1.0% of stake and an a0 of 1.0 would require all pools to be 100.0% pledged.
-
-<img src="new equation.png" width="400">
+The proposed reward retains the function of $k$ for limiting rewards based on stake but repurposes the $a0$ parameter for enforcing reward limits based on pledge leverage. The equation equally balances both reward parameters. Instead of $a0$ ranging from 0.0 to infinity the $a0$ parameter is intended to range from 10,000.0 down to 1.0. An $a0$ value of 100.0 would require pools to pledge 1.0% of stake and an $a0$ of 1.0 would require all pools to be 100.0% pledged.
 
-R = ( reserve * rho + fees )( 1.0 - tau )
+$$ R = ( reserve * \rho + fees )( 1 - \tau ) $$
 
-r( sigma, lambda ) = R * min{ sigma, a0 * lambda, 1/k }
+$$ r( \sigma, \lambda ) = R * min\bigg\{ \sigma, a0 * \lambda, \frac{1}{k} \bigg\} $$
 
 
 The new equation is computationally simple and purposefully does not use logarithms, exponents, or geometric curves. Instead of an incentive based tradeoff between egalitarian rewards and a perceived Sybil resilience the new equation enforces both egalitarian rewards and pledge-based Sybil resilience. A simple flat egalitarian yield ceiling with pledge leverage enforcement for Sybil defense has a profound psychological effect: Stakeholders know there is no way to game the system for yield, either individually or collectively with governance, and pledge is absolutely mandatory. Without any engineered bias Cardano decentralization would converge to the diversity of the underlying community, services, and stakeholder distribution. If this proposal is eventually adopted changes in community diversity, not changes in a formula or parameters, would change decentralization.
 
 
 # Rationale
 
-## Recast of a0
+## Recast of $a0$
```

Comment:
I don't think it renders in the headings.

Reply:
It's subtle, i did some back and forth testing, but can none-the-less remove because it's basically no different.

Reply:
removed